movieApp/AllMoviesSectionCell.swift

```diff
@@ -0,0 +1,132 @@
+////
+////  AllMoviesSectionCell.swift
+////  movieApp
+////
+////  Created by <3 on 18.05.2024..
+////
+//
+
+
+
+
+
```

Comment:
To further on clean your code, you can remove all of these headers from your .swift files. They are autogenerated and don't really serve any purpose. Good rule of thumb is to start each of your .swift files with and "import" statement, for instance 

```
import UIKit
import MovieAppData
```